src/eei.cpp

```diff
@@ -191,6 +191,35 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
       return Literal();
     }
 
+    if (import->base == Name("create")) {
+      cout << "create" << endl;
+
+      uint32_t valueOffset = arguments[0].geti32();
+      uint32_t dataOffset = arguments[1].geti32();
+      uint32_t length = arguments[2].geti32();
+      uint32_t resultOffset = arguments[3].geti32();
+
+      evm_message create_message;
+      evm_result create_result;
+      vector<uint8_t> contract_code(length);
+      
+      create_message.address = {  };
+      create_message.sender = msg.address;
+      create_message.value = loadUint128(valueOffset);
+      loadMemory(dataOffset, contract_code, length);
+      create_message.input = contract_code.data();
+      create_message.input_size = length;
+      create_message.code_hash = { };
+      create_message.depth = msg.depth + 1;
+      create_message.kind = EVM_CREATE;
+      create_message.flags = 0;
+
```

Comment:
`gas` must be calculated properly here according to the weird rules we have ...

`gasLeft - (gasLeft / 64)`